cruise-control/src/test/java/com/linkedin/kafka/cruisecontrol/common/DeterministicCluster.java

```diff
@@ -503,4 +509,31 @@ public static ClusterModel getHomogeneousCluster(Map<Integer, Integer> rackByBro
         (key, value) -> cluster.createBroker(value.toString(), Integer.toString(key), key, commonBrokerCapacityInfo, diskCapacityByLogDir != null));
     return cluster;
   }
+
+  /**
+   * Generate the cluster metadata from given cluster model.
+   * @param clusterModel The cluster model.
+   * @return The cluster metadata.
+   */
+  public static Cluster generateClusterFromClusterModel(ClusterModel clusterModel) {
+    Map<Integer, Node> nodes = new HashMap<>();
+    clusterModel.brokers().forEach(b -> nodes.put(b.id(), new Node(b.id(), b.host().toString(), 0, b.rack().id())));
```

Comment:
Rather than `b.host().toString()`, maybe it is more preferable to use `host().name()`?
For historical reasons, `toString()` method is returning `XML` format -- which is something we should change in this or another patch.